async_simple/FutureState.h

```diff
@@ -168,8 +168,8 @@ class FutureState {
     }
 
 public:
-    Try<T>& getTry() noexcept { return _try; }
-    const Try<T>& getTry() const noexcept { return _try; }
+    Try<T>& getTry() noexcept { return _try_value; }
```

Comment:
Why the renaming is necessary?

Reply:
Can't use _try or __try variable in windows, will cause confilict:https://godbolt.org/z/KcYMsTvx5